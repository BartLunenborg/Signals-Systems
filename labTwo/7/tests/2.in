0.9
3: [17,2,19]
10: [1,6,12,11,19,4,21,12,3,9]
