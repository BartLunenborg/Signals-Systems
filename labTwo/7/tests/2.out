4 1.00000
7 0.90419
